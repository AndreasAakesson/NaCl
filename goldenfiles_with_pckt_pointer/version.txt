NaCl commit: 4d04cb8
(November 16 2017)
