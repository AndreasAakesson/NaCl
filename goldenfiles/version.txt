NaCl commit: 571262a
(November 8 2017)
